pom.xml

```
@@ -70,6 +70,7 @@
     <bnd.version>6.1.0</bnd.version>
     <eea.version>2.3.0</eea.version>
     <karaf.version>4.3.4</karaf.version>
+    <ohc.version>3.2.0-SNAPSHOT</ohc.version>
```

I probably need to update this soon. 

```suggestion
    <ohc.version>3.3.0-SNAPSHOT</ohc.version>
```

---

I'm assuming you're planning a merge after the release then?

---

Yes I think this is something for OH 3.3.0.